workspaces/global-header/plugins/global-header/src/components/HeaderIconButtonComponent/HeaderIconButton.tsx

```diff
@@ -16,25 +16,34 @@
 
 import React from 'react';
 import IconButton from '@mui/material/IconButton';
-import SmallIconWrapper, { IconComponentType } from './SmallIconWrapper';
+import Tooltip from '@mui/material/Tooltip';
+import { HeaderIcon } from './HeaderIcon';
 
-interface HeaderIconButtonProps {
-  Icon: IconComponentType;
+/**
+ * @public
+ * Header Icon Button properties
```

Comment:
nit: Documentation first, then the tags :)

```suggestion
 * Header Icon Button properties
 * @public
```